### PR TITLE
Update Developer Guide Use Case 1-4

### PR DESCRIPTION
**Pull request by akhilvuputuri**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -321,30 +321,184 @@ _{More to be added}_
 === Diet Tracker
 
 [discrete]
-=== Use case: Add Person
+=== Use case UC01: Update User Profile
```
Comment (sheexiong) [inline docs/DeveloperGuide.adoc]: Should use Use case: UC01 Update User Profile instead.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -321,30 +321,184 @@ _{More to be added}_
...
+
+[none]
+* 1a. List command contains invalid flag.
+** 1a1. Diet Tracker prints out default list which contains food logged for the day
+
+* 3a. Invalid or empty index keyed into command.
+** 3a1. Diet Tracker requests user to re-enter index in correct format
+** 3a2. User enters delete item in  correct format +
+Steps 3a1-3a2 are repeated until the correct  entered is correct. +
+Use ends.
```
Comment (ailanthustng) [inline docs/DeveloperGuide.adoc]: "Use case ends"

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -321,30 +321,184 @@ _{More to be added}_
...
+
+* 3b. No additional tags and data keyed in as flags to replace existing data.
+** 3b1. Diet Tracker requests user to re-enter command with at least one flag
+** 3a2. User enters command with flag and data +
+Steps 3b1-3b2 are repeated until the correct  entered is correct. +
+Use ends.
```
Comment (ailanthustng) [inline docs/DeveloperGuide.adoc]: "Use case ends"

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -321,30 +321,184 @@ _{More to be added}_
...
+Steps 3a1-3a2 are repeated until the correct  entered is correct. +
+Use ends.
+
+[discrete]
+=== Use case UC04: Edit Food Item
+
+*MSS*
+
+1.  User lists out existing items
+2.  Diet Tracker shows the list based on flags entered
```
Comment (ailanthustng) [inline docs/DeveloperGuide.adoc]: Please change "Diet Tracker" to "EYLAH"

**Pull request merged**